datacontract/imports/glue_importer.py

```diff
@@ -140,8 +139,13 @@ def import_glue(
     if table_names is None:
         table_names = get_glue_tables(source)
 
+    server_kwargs = {"type": "glue", "account": catalogid, "database": source}
+
+    if location_uri:
+        server_kwargs["location"] = location_uri
```

Comment:
`location_uri` with `"None"` string is considered to be None value while desearalizing to YAML.
Therefore Glue database without a location creates invalid data contract:
```
datacontract-cli git:(56f4dbe) ✗ datacontract test test1.yaml 
Testing test1.yaml
WARNING:root:Data Contract YAML is invalid. Validation error: data.servers.production must be valid exactly by one definition (0 matches found)
ERROR:root:Run operation failed: [lint] Check that data contract YAML is valid - None - failed - data.servers.production must be valid exactly by one definition (0 matches found) - datacontract
╭────────┬────────────────────────────────────────┬───────┬────────────────────────────────────────────────────╮
│ Result │ Check                                  │ Field │ Details                                            │
├────────┼────────────────────────────────────────┼───────┼────────────────────────────────────────────────────┤
│ failed │ Check that data contract YAML is valid │       │ data.servers.production must be valid exactly by   │
│        │                                        │       │ one definition (0 matches found)                   │
╰────────┴────────────────────────────────────────┴───────┴────────────────────────────────────────────────────╯
🔴 data contract is invalid, found the following errors:
1) data.servers.production must be valid exactly by one definition (0 matches found)
```
Let's ignore destination param if it's not specified

Reply:
Is the location optional in Glue?

Reply:
@jochenchrist wow, thanks for the fast feedback! Yes, it's optional.
That's the location description you get in their UI:
![image](https://github.com/user-attachments/assets/43f1ae25-0646-47d0-8e7b-c60b3dd11507)
and confirming I do have some Glue databases without a location specified

Reply:
OK, do you still get this issue? `location` was already optional in the schema.
If you still have this error, can you post the server object that you get.

Reply:
@jochenchrist yup, it's optional in Pydantic model. However, the JSON schema of specification does not allow anything except URI string: https://github.com/datacontract/datacontract-specification/blob/816e6ae21e7ebdd2b9261734bd1a7ee47388574c/datacontract.schema.json#L261
Therefore, Glue database without location will create an invalid contract with `location: None`.
If you validate this contract with `datacontract test` you will get the error above

Reply:
@jochenchrist yup, it's optional in Pydantic model. However, the JSON schema of specification does not allow anything except URI string: https://github.com/datacontract/datacontract-specification/blob/816e6ae21e7ebdd2b9261734bd1a7ee47388574c/datacontract.schema.json#L446
Therefore, Glue database without location will create an invalid contract with `location: None`.
If you validate this contract with `datacontract test` you will get the error above.
We can make the

Reply:
If the location is not set, then the data contract YAML should be valid (location is not a required property):
```
servers:
  production:
    type: glue
    account: '123456789012'
    database: test_database
```

Reply:
@roykoand Yes, sorry for the confusion. It's fixed by this PR but was not working before. Thanks!